<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playerControlWidget</class>
 <widget class="QWidget" name="playerControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(253, 253, 253);
background-color: rgb(255, 251, 251);
border-radius:8px;
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="pushButton_songImage">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="myResources.qrc">
           <normaloff>:/player/images/player/pretty_crazy.jpg</normaloff>:/player/images/player/pretty_crazy.jpg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>70</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_songName">
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
background: transparent;</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;天之骄女&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_singerName">
              <property name="styleSheet">
               <string notr="true">color: rgb(100, 100, 100);
background: transparent;</string>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;-&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt; 容祖儿&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_like">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">/* 播放按钮 */
QPushButton {
    /* 替换 background-image */
    border-image: url(:/player/images/player/like.png); 
    qproperty-cursor: pointinghand; /* 鼠标悬停变手掌 */
    border: none; /* 确保没有边框，否则 border-image 可能会受到影响 */
    background-color: transparent; /* 保持透明背景色 */
}

QPushButton:hover
 {  
	border-image: url(:/player/images/player/like1.png); 
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_commend">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">/* 播放按钮 */
QPushButton {
    /* 替换 background-image */
    border-image: url(:/player/images/player/comment.png); 
    qproperty-cursor: pointinghand; /* 鼠标悬停变手掌 */
    border: none; /* 确保没有边框，否则 border-image 可能会受到影响 */
    background-color: transparent; /* 保持透明背景色 */
}

QPushButton:hover, 
QPushButton:pressed {    
border-image: url(:/player/images/player/comment_press.png); 
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_more">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">/* 播放按钮 */
QPushButton {
    /* 替换 background-image */
    border-image: url(:/player/images/player/more1.png); 
    qproperty-cursor: pointinghand; /* 鼠标悬停变手掌 */
    border: none; /* 确保没有边框，否则 border-image 可能会受到影响 */
    background-color: transparent; /* 保持透明背景色 */
}

QPushButton:hover, 
QPushButton:pressed {    
border-image: url(:/player/images/player/more2.png); 
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_16">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="100,50,50,1">
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,10,1,10,1,10,1,10,1,10,10">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_playOrder">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    /* 替换 background-image */
    border-image: url(:/player/images/player/playmodel10.png);
    qproperty-cursor: pointinghand; /* 鼠标悬停变手掌 */
    border: none; /* 确保没有边框，否则 border-image 可能会受到影响 */
    background-color: transparent; /* 保持透明背景色 */
}
QPushButton:hover {
border-image: url(:/player/images/player/playmodel11.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_play_prev">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">/* 播放按钮 */
QPushButton {
    /* 替换 background-image */
    border-image: url(:/player/images/player/btn_playPrev_100.png); 
    qproperty-cursor: pointinghand; /* 鼠标悬停变手掌 */
    border: none; /* 确保没有边框，否则 border-image 可能会受到影响 */
    background-color: transparent; /* 保持透明背景色 */
}

QPushButton:hover, 
QPushButton:pressed {    border-image: url(:/player/images/player/btn_playPrev_50.png); 
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_play">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">/* 播放按钮 */
QPushButton {
    /* 替换 background-image */
    border-image: url(:/player/images/player/btn_pausing.png); 
    qproperty-cursor: pointinghand; /* 鼠标悬停变手掌 */
    border: none; /* 确保没有边框，否则 border-image 可能会受到影响 */
    background-color: transparent; /* 保持透明背景色 */
}
QPushButton:checked {
border-image: url(:/player/images/player/btn_playing.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_play_next">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">/* 播放按钮 */
QPushButton {
    /* 替换 background-image */
    border-image: url(:/player/images/player/btn_playNext_100.png); 
    qproperty-cursor: pointinghand; /* 鼠标悬停变手掌 */
    border: none; /* 确保没有边框，否则 border-image 可能会受到影响 */
    background-color: transparent; /* 保持透明背景色 */
}

QPushButton:hover, 
QPushButton:pressed {    border-image: url(:/player/images/player/btn_playNext_50.png);
cursor: pointinghand; 
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_volume">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">/* 播放按钮 */
QPushButton {
    /* 替换 background-image */
    border-image: url(:/player/images/player/volume.png); 
    qproperty-cursor: pointinghand; /* 鼠标悬停变手掌 */
    border: none; /* 确保没有边框，否则 border-image 可能会受到影响 */
    background-color: transparent; /* 保持透明背景色 */
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,100,1">
         <item>
          <widget class="QLabel" name="label_nowtime">
           <property name="styleSheet">
            <string notr="true">color: rgb(100, 100, 100);
background: transparent;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;00：00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="mySlider" name="slider_progress">
           <property name="styleSheet">
            <string notr="true">        QSlider::groove:horizontal {
            height: 6px;               /* 进度条高度 */
            /*background: #c0c0c0;       背景色 */
            border-radius: 3px;        /* 圆角 */
background: transparent;
        }
        QSlider::handle:horizontal {
    width: 14px;                   /* 滑块宽度 */
    height: 14px;                  /* 滑块高度，保持等于宽度 */
    margin: -4px 0;                /* 让滑块居中在槽上 */
    border-radius: 7px;            /* 半宽/半高 =&gt; 圆形 */
    background: #000000;           /* 滑块颜色 */
    border: 2px solid #999999;     /* 滑块边框 */
        }
        QSlider::sub-page:horizontal {
            background: #000000;       /* 已播放部分颜色 */
            border-radius: 3px;
        }
        QSlider::add-page:horizontal {
            background: #DEDEDE;       /* 未播放部分颜色 */
            border-radius: 3px;
        }
        QSlider::handle:horizontal:hover {
    background: #3d9eff;
    border: 2px solid #3d9eff;
        }
background: transparent;</string>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_totaltime">
           <property name="styleSheet">
            <string notr="true">color: rgb(100, 100, 100);
background: transparent;</string>
           </property>
           <property name="text">
            <string>00：00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_14">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="140,60,300">
       <item>
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="pushButton_mv">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">/* 播放按钮 */
QPushButton {
    /* 替换 background-image */
    border-image: url(:/player/images/player/MV.png); 
    qproperty-cursor: pointinghand; /* 鼠标悬停变手掌 */
    border: none; /* 确保没有边框，否则 border-image 可能会受到影响 */
    background-color: transparent; /* 保持透明背景色 */
}
QPushButton:hover {
border-image: url(:/player/images/player/MV_press.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_playlist">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">/* 播放按钮 */
QPushButton {
    /* 替换 background-image */
    border-image: url(:/player/images/player/playList.png); 
    qproperty-cursor: pointinghand; /* 鼠标悬停变手掌 */
    border: none; /* 确保没有边框，否则 border-image 可能会受到影响 */
    background-color: transparent; /* 保持透明背景色 */
}
QPushButton:hover {
border-image: url(:/player/images/player/playList_press.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_10">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mySlider</class>
   <extends>QSlider</extends>
   <header>myslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="myResources.qrc"/>
 </resources>
 <connections/>
</ui>
